index.js

```diff
@@ -0,0 +1 @@
+console.log('Hello world')
```

Comment:
Always use semicolon:

```suggestion
console.log('Hello world');
```